istioctl/cmd/testdata/webhook/example-mutating-webhook-config.yaml

```diff
@@ -26,3 +26,4 @@ webhooks:
         resources:
           - pods
     sideEffects: Unknown
+    admissionReviewVersions: ["v1beta1", "v1"]
```

Comment:
Shouldn't make "v1" first?

	// AdmissionReviewVersions is an ordered list of preferred `AdmissionReview`
	// versions the Webhook expects. API server will try to use first version in
	// the list which it supports.

Reply:
We are using v1beta1 apis in the go code

Reply:
What do you mean by `go code`? As i know, this admission review is originated from kube-apiserver, if specify `admissionReviewVersions` in the webhookconfiguration, it will try the version in order.

Reply:
And if so, why do even need the adapter?

Reply:
Since k8s 1.16
https://github.com/kubernetes/kubernetes/blob/d2adc15fc357632eeb65c75288fb46a910ca05eb/staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/request/admissionreview.go#L130-L148

Kube-apiserver call admission with the first version specified in `admissionReviewVersions`. This is not good, and it leaves all the touch work to the developer working on admission webhook.